### ISSUE TITLE
Request : Expose info about jobs

### ISSUE THREAD

**Issue by krontogiannis** [mbraceproject/MBrace.Core#125]
It would be nice to expose information about jobs per cloudtask (`CloudTask.ShowJobInfo` or sth). See [1](https://github.com/krontogiannis/MBrace.Azure/blob/thesis/src/MBrace.Azure.Runtime/RuntimeInfo/JobInfo.fs#L250) and [2](https://github.com/krontogiannis/MBrace.Azure/blob/master/src/MBrace.Azure.Runtime/Runtime/JobManager/JobRecord.fs#L29) as an example. `IJobQueue` should support this (and perhaps be renamed to `IJobManager`).

**Comment by eiriktsarpalis** [comment]
Agreed. In general I think we need to include support for task and job hierarchies which is something currently missing.
